packages/platform/platform-cache/src/services/PlatformCache.ts

```diff
@@ -78,8 +79,9 @@ export class PlatformCache {
     return deserialize(this.cache?.get<T>(key), options);
   }
 
-  async set<T>(key: string, value: any, options?: CachingConfig): Promise<T | undefined> {
-    return this.cache?.set<T>(key, value, options);
+  async set<T>(key: string, value: any, options?: CachingConfig<T>): Promise<T | undefined> {
+    this.cache?.set(key, value, options?.ttl);
```

Comment:
`set` can return a promise:
```suggestion
    await this.cache?.set(key, value, options?.ttl);
```

Reply:
true!

Reply:
my bad ^^